src/runtime/container.cc

```diff
@@ -49,6 +49,23 @@ TVM_REGISTER_GLOBAL("runtime.Array").set_body([](TVMArgs args, TVMRetValue* ret)
   *ret = Array<ObjectRef>(data);
 });
 
+// Concatenate n TVMArrays
+TVM_REGISTER_GLOBAL("runtime.ArrayConcat").set_body([](TVMArgs args, TVMRetValue* ret) {
```

Comment:
given this is not as a commonly used function, let us move it to https://github.com/apache/tvm/blob/main/web/emcc/wasm_runtime.cc

Reply:
Makes sense; just moved and renamed to `tvmjs.runtime.ArrayConcat`.